### DISCUSSION TITLE
Unable to import custom content

### DISCUSSION THREAD

**Discussion by derek-jackson** [mozilla/hubs#5967] [Hubs Cloud Questions] [answered]
Hi, I've created some custom Avatar GLB files. Spoke loads them fine, but when I try to paste a URL into the hubs Cloud 'Import Content' interface, it just sits there spinning. I've tried the Spoke URL and our own webspace but neither seems to work. We only set the account up yesterday - is there something I'm missing?

![image](https://user-images.githubusercontent.com/36922016/221161895-5e1c8327-4e8b-43a3-b096-e4a043d5e29f.png)

**Comment by rawnsley** [comment, marked as answer]
I think they have to be avatar pages on Mozilla Hubs or your own Hubs Cloud rather than GLB files.

**Reply by derek-jackson** [reply]
Thanks. Hadn't appreciated it'd be a link to the Avatar's page rather than the model data.
